package.json

```diff
@@ -7,20 +7,22 @@
     "mysql": "^2.11.1"
   },
   "devDependencies": {
-    "babel": "^6.1.18",
+    "babel": "^6.23.0",
+    "babel-cli": "^6.24.1",
```

Comment:
actually I would like to suggest to move global deps to dev deps so we utilize usage of npm scripts instead of having global dependencies when contributing to this project.